### PR TITLE
fix: remove cache for chain list

### LINKED ISSUE #5373: Caching in KodaDot
Apart from running issues, I would like to see remove all burdens we have with cache as it's bit breaking experience for existing users and future users who won't see our updates. There are inconsistencies between prod and beta even shortly after release. 
I sense PWA plugin we are using is doing bad things plus I've missed on sidebar don't see same state as I see on prod, i.e. on beta I see TNKR w/o any balances at KSM and TNKR, at prod I don't see TNR and see balances

---
Hey, on beta I still don't see rmrk2 and on production i don't see TNKR
- #5372 

telegram-cloud-photo-size-4-5834626949388024482-y

With fix at on mobile I don't even see settings so... and doubt users would need to go everytime to settings we will push new version. 

- #5356 

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/5326#issuecomment-1483828071_

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5373
- [x] get chain list from static instead of localStorage (vuex-persist)

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: composables/useChain.ts [hunk 1]
```diff
@@ -24,15 +25,15 @@ export default function () {
...
 
   const availableChains = computed(() => {
-    const availableUrlPrefixes = $store.getters['availableUrlPrefixes']
+    const chainList = availablePrefixes()
 
     if (!$config.public.dev) {
-      return availableUrlPrefixes.filter(
-        (urlPrefix) => !getChainTestList().includes(urlPrefix.value)
+      return chainList.filter(
+        (urlPrefix) => !getChainTestList().includes(`${urlPrefix.value}`)
```
Comment (roiLeo) [inline composables/useChain.ts:32]:
```suggestion
        (urlPrefix) => !getChainTestList().includes(urlPrefix.value)
```

**Review by roiLeo — APPROVED** [review]

File: composables/useChain.ts (hunk 1 above)
Comment (roiLeo) [reply composables/useChain.ts:32]: yep, it appeared a few minutes later. 
maybe wrap urlPrefix.value in a `String()` or type it `as string`

File: utils/chain.ts [hunk 2]
```diff
@@ -81,3 +83,31 @@ export const getChainNameByPrefix = (prefix: string) => {
...
+  rmrk2: 'rmrk',
+}
+
+export const availablePrefixes = (): Option[] => {
+  return chainPrefixes.map((prefix) => ({
+    info: infos[prefix],
+    text: NAMES[prefix],
+    value: prefix === 'ksm' ? 'rmrk' : prefix,
+  }))
+}
```
Comment (vikiival) [inline utils/chain.ts:113]: so let's move it first?

File: composables/useChain.ts [hunk 3]
```diff
@@ -24,15 +25,15 @@ export default function () {
...
-    const availableUrlPrefixes = $store.getters['availableUrlPrefixes']
+    const chainList = availablePrefixes()
 
     if (!$config.public.dev) {
-      return availableUrlPrefixes.filter(
-        (urlPrefix) => !getChainTestList().includes(urlPrefix.value)
+      return chainList.filter(
+        (urlPrefix) => !getChainTestList().includes(String(urlPrefix.value))
       )
     }
```
Comment (vikiival) [inline composables/useChain.ts:34]: Isnt it better to hardcode prod chain list in `utils/chain.ts` ?

**Review by vikiival — CHANGES_REQUESTED** [review]
Let's close issue

- https://github.com/kodadot/packages/issues/112

First

**Comment by vikiival** [comment]
Otherwise, Really nice job !

**Comment by vikiival** [comment]
Or lets merge better done than perfect

**Pull request merged**